tests/testthat/test-bind_tables.R

```diff
@@ -0,0 +1,49 @@
+# Load initial variables
```

Comment:
This test suite is great, but instead of relying on `read_redcap_tidy()` to create the data set to test, it would be better to load a saved .rds file that contains the output of `read_redcap_tidy()`.

Reply:
Done 👍